python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -41,6 +41,10 @@ class FitsRawFormatterBase(FitsExposureFormatter, metaclass=ABCMeta):
     FITS files.
     """
 
+    # This has to be explicit until we fix camera geometry in RFC-605
```

Comment:
So is this a workaround for DM-26327? Should that ticket be mentioned?

Reply:
DM-20746 I think. I was continuing the DECam trend of referring to the RFC and not the ticket. When 20746 is done this will all have to be rejigged a bit.

Reply:
Yes, sorry, looked at the wrong tab 🤦.